src/app/components/PlayerPanel.ts

```diff
@@ -0,0 +1,132 @@
+import { Player } from '../../Player';
+import { Render } from '../utils/Render';
+import { View } from '../View';
+import { flatten } from 'lodash';
+import { AnimalNames } from '../../Enums/AnimalNamesEnum';
+
+export class PlayerPanel {
+  /**
+   * Creates PlayerPanel based on data given
+   * @param player accepts instance of current player
+   * @param view accepts instance of View componnet
+   */
+  constructor(private player: Player, private view: View) {}
+
+  createPlayerPanel(): HTMLElement {
+    return Render.elementFactory(
+      'div',
+      {
+        className: 'player-panel',
+        style: `background-color: ${this.player.theColor};`,
+      },
+      this.createPlayerBoard(),
+      this.createResultWindow(),
+      this.createButtonPanel(),
+    );
+  }
+
+  private createPlayerBoard(): HTMLElement {
+    return Render.elementFactory(
+      'div',
+      {
+        className: 'player-panel__board',
+      },
+      Render.elementFactory(
+        'div',
+        { className: 'player-panel__info' },
```

Comment:
I think class names should be consistent between player-panel_board and player__board, perhaps just with some changes? If you use PlayerBoard (after it is merged) these won't be an issue, since I imagine you would use PlayerBoard?

Reply:
I'm not sure if I will use it, because player board will possibly look different than panel. But this will be reconsidered when we get feedback from client.